operator/internal/manifests/internal/config/loki-config.yaml

```diff
@@ -5,6 +5,19 @@ chunk_store_config:
     enable_fifocache: true
     fifocache:
       max_size_bytes: 500MB
+ruler:
+  storage:
+    type: local
+    local:
+      directory: /etc/loki/rules
+  ring:
+    kvstore:
+      store: memberlist
+  rule_path: /tmp/loki/scratch
+  remote_write:
```

Comment:
Please add if statement here that checks that `.Prometheus.Endpoint` is not empty. You can look https://pkg.go.dev/text/template for the syntax. Without that, if the `url` is empty there is no deployment

Reply:
WAL Storage should be added here.

Reply:
What do you mean by WAL Storage @Red-GV